array.go

```diff
@@ -746,6 +746,128 @@ func (a *Array) Equal(value interface{}) *Array {
 	return a.IsEqual(value)
 }
 
+// InList succeeds if array is listed by given [values...].
```

Comment:
Here and in other methods, I suggest to reword this part a bit:

```
InList succeeds if whole array is equal to one of the elements from given list of arrays.
```

Reply:
Addressed in `Array` and others as well.